Monika After Story/game/script-topics.rpy

```diff
@@ -12030,3 +12030,44 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_relationcomfort",
+            category=["romance","advice"],
+            prompt="Comfortable in a relationship",
+            random=True
+        )
+    )
+
+label monika_relationcomfort:
+    m 1wud "You know [player], I've been thinking about our relationship lately..."
```

Comment:
Maybe it's just me, but I think that's quite an expression to lead off with, potentially out of nowhere.

Reply:
It is, but it's purposefully set up to be so. It's supposed to show she's thought about it pretty deeply, moreso than a standard expression conveys